spec/features/access_etd_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+include Warden::Test::Helpers
+
+RSpec.feature 'Access an Etd', js: false do
+  let(:etd)         { FactoryGirl.create(:public_etd, pdf: pdf) }
+  let(:pdf)         { FactoryGirl.create(:pdf_upload) }
+  let(:private_etd) { FactoryGirl.create(:etd, pdf: pdf) }
+
+  scenario 'searching submitted ETDs' do
+    visit '/'
+
+    fill_in 'Search Hyrax', with: etd.first_title
+    click_button 'Go'
+
+    within(:css, "li#document_#{etd.id}") do
+      expect(page).to have_content etd.first_title
+    end
+  end
+
+  context 'with ingested file', :perform_enqueued do
+    # Only enqueue the ingest job, not charactarization.
+    before { ActiveJob::Base.queue_adapter.filter = [IngestJob] }
+
+    scenario 'downloading' do
+      visit      "concern/etds/#{etd.id}"
+      click_link "Download \"#{etd.representative.first_title}\""
+
+      expect(page.response_headers['Content-Disposition']).to include 'attachment'
+    end
+
+    scenario 'downloading restricted' do
+      visit "concern/etds/#{private_etd.id}"
+      expect(page).not_to have_content 'Download'
```

Comment:
Is this a sufficient expectation.  Not only should I not have a download link, in the default configuration I should get "You are not authorized to access this page."

Reply:
> "You are not authorized to access this page."

You currently don't get a message like this for non-public items. I think this is also for discussion in #87.